src/SprykerEco/Yves/Amazonpay/Theme/default/layout.twig

```diff
@@ -23,5 +23,6 @@
 {% endblock %}
 
 {% block javascript %}
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
We have jquery in demoshop. Generally vendor javascript shold be included with package.json